docs/pages/deploy-a-cluster/high-availability.mdx

```diff
@@ -126,24 +126,28 @@ events:
 - Google Cloud Firestore
 - etcd (cluster state only)
 
-For Amazon DynamoDB and Google Cloud Firestore, your Teleport configuration (which
-we will describe in more detail in the [Configuration](#configuration) section)
-names a table or collection where Teleport stores cluster state and audit
-events. 
-
-The Teleport Auth Service manages the creation of any required DynamoDB tables
-or Firestore collections itself, and does not require them to exist in advance.
+For Amazon DynamoDB and Google Cloud Firestore, your Teleport configuration
+(which we will describe in more detail in the [Configuration](#configuration)
+section) names a table or collection where Teleport stores cluster state and
+audit events. 
```

Comment:
It might be worth clarifying here that the tables for cluster state and audit events are always separate, as they have different schemas. I know we do mention this specifically in the partial, but I'm a fan of overstating things like this to make sure people are aware.